_includes/locale/locale.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+  const locales = [
+    'en'    => 'English', // Default
+    'es-ES' => 'Española',
+    'fr-FR' => 'Français'
+  ];
```

Comment:
Currently not used. Maybe when we settle on UX for specifying UI locale on the site...